Cargo.lock

```diff
@@ -2093,6 +2093,12 @@ dependencies = [
  "libc",
 ]
 
+[[package]]
+name = "hermit-abi"
```

Comment:
I'm actually quite interested in why this package ended up in here. That seems strange.

Reply:
Ah, it looks like it's a dependency of `is_terminal` and `num_cpus`. In that case I don't think we need to worry about it.